cmd/describe/machinepool/cmd.go

```diff
@@ -71,6 +71,7 @@ func DescribeMachinePoolRunner(userOptions DescribeMachinepoolUserOptions) rosa.
 			userOptions.machinepool = argv[0]
 		} else {
 			err := cmd.ParseFlags(argv)
+			userOptions.machinepool = cmd.Flag("machinepool").Value.String()
```

Comment:
@hunterkepley Is `userOptions.machinepool` not set at the entry to this function if you pass the `--machinepool` argument to this command? It should be given the way you initialise the `FlagSet` in the `NewDescribeMachinePool` function